frontend/src/utils/maxLengths.ts

```diff
@@ -0,0 +1,11 @@
+export const MaxLengths = {
+	email: 200,
+	password: 100,
+	name: 255,
+	local: 300,
+	geral: 200,
```

Comment:
```suggestion
	general: 200,
```

Opcional, mas acho que assim fica melhor